crates/conformance-test-utils/src/hive/genesis.rs

```diff
@@ -23,6 +36,125 @@ impl HiveGenesisConfig {
+    let mut hive_accounts: Vec<(reth_primitives::H160, AccountInfo)> = hive_genesis.alloc.into_iter().collect();
+    hive_accounts.sort_by_key(|(address, _)| *address);
+    hive_accounts.iter().for_each(|(evm_address, account_info)| {
+        // Use the given Kakarot contract address and declared proxy class hash for compute_starknet_address
+        let starknet_address = compute_starknet_address(
+            kakarot_address,
+            account_proxy_class_hash,
+            FieldElement::from_byte_slice_be(evm_address.as_bytes()).unwrap(), /* safe unwrap since evm_address
+                                                                                * is 20 bytes */
+        );
+        // Push to contracts
+        loader.contracts.push((HexFelt(starknet_address), HexFelt(account_proxy_class_hash)));
+
+        // Set the balance of the account
+        // Call genesis_fund_starknet_address util to get the storage tuples
+        let balance_storage_tuples = genesis_fund_starknet_address(starknet_address, account_info.balance);
+        balance_storage_tuples.iter().for_each(|balance_storage_tuple| {
+            loader.storage.push(unsafe {
+                std::mem::transmute::<((Felt, Felt), Felt), ((HexFelt, HexFelt), HexFelt)>(*balance_storage_tuple)
+            });
+        });
+
+        // Set the storage of the account, if any
+        if let Some(storage) = account_info.storage.as_ref() {
+            let mut storage: Vec<(U256, U256)> = storage.iter().map(|(k, v)| (*k, *v)).collect();
+            storage.sort_by_key(|(key, _)| *key);
+            storage.iter().for_each(|(key, value)| {
+                // Call genesis_set_storage_kakarot_contract_account util to get the storage tuples
+                let storage_tuples = genesis_set_storage_kakarot_contract_account(starknet_address, *key, *value);
+                storage_tuples.iter().for_each(|storage_tuples| {
+                    loader.storage.push(unsafe {
+                        std::mem::transmute::<((Felt, Felt), Felt), ((HexFelt, HexFelt), HexFelt)>(*storage_tuples)
+                    });
+                });
+            });
+        }
+
+        // Set the bytecode of the account, if any
+        if let Some(bytecode) = account_info.code.as_ref() {
+            // Call genesis_set_code_kakarot_contract_account util to get the storage tuples
+            let code_storage_tuples = genesis_set_bytecode(bytecode, starknet_address);
+            code_storage_tuples.iter().for_each(|code_storage_tuple| {
+                loader.storage.push(unsafe {
+                    std::mem::transmute::<((Felt, Felt), Felt), ((HexFelt, HexFelt), HexFelt)>(*code_storage_tuple)
+                });
+            });
+        }
+    });
+
+    // Serialize the loader to a string
+    let madara_genesis_str = serde_json::to_string_pretty(&loader)?;
+    // Write the string to a file
+    fs::write(Path::new("src/hive/madara_genesis.json"), madara_genesis_str)?;
```

Comment:
Would avoid hardcoding this path as well. Best would be to be able to set the output path when calling the function, what do you think?